### DISCUSSION TITLE
ScrollBar Extends into Headers and Footers

### DISCUSSION THREAD

**Discussion by blentz100** [icflorescu/mantine-datatable#575] [General]
**Describe the bug**
I noticed when I upgraded to v7 that the scrollbars extend into the column headers and footers for my app. I'm not sure if I'm missing something but this didn't happen in v6. 

**To Reproduce**

1. You can see this in the docs: https://icflorescu.github.io/mantine-datatable/examples/scrollable-vs-auto-height/
2. Scroll up and down and you'll see the scroll bar extending into the header.
3. You can also see the issue on my app here on this Vercel Preview Deployment: https://homepage-1rl4rp0fj-brendanlentz-gmailcom.vercel.app/
4. Scroll up and down to see the scroll bar extending into the header and footer
5. See error

**Expected behavior**
1. You can see here in the v6 docs that the scrollbar does not extend into the header: https://icflorescu.github.io/mantine-datatable-v6/examples/scrollable-vs-auto-height
2. Scroll bar should not extend into header and footer area
3. You can also see that the scrollbars here in the Vercel preview deployment do not extend into the header or footer: https://homepage-6judybk4b-brendanlentz-gmailcom.vercel.app/

**Screenshots**

**Desktop (please complete the following information):**

- OS: M1 Air Sonoma 14.0
- Browser Brave 
- Version 1.58.135

**Smartphone (please complete the following information):**

**Additional context**
@icflorescu Thanks for all the amazing work you do on this library!

**Comment by icflorescu** [comment]
Yes, this behaviour has indeed changed since v6.
If you ask me, I'd say the new behaviour is more consistent with what we have in browsers when we use "fixed headers" - the browser scrollbar spans across the entire height of the page.

I'll convert this to a discussion, since it's not exactly a bug or an issue, but rather a matter of opinion and/or personal preferences...
